python/dataproc_templates/jdbc/README.md

```diff
@@ -249,4 +249,197 @@ export JARS="gs://my-gcp-proj/jars/mysql-connector-java-8.0.29.jar,gs://my-gcp-p
 --jdbctojdbc.output.table="employees_out" \
 --jdbctojdbc.output.mode="overwrite" \
 --jdbctojdbc.output.batch.size="1000"
+```
+
+# 2. JDBC To GCS
+
+Template for reading data from JDBC table and writing into files in Google Cloud Storage. It supports reading partition tabels and supports writing in JSON, CSV, Parquet and Avro formats.
+
+## Arguments
+
+* `jdbctogcs.input.url`: JDBC input URL
+* `jdbctogcs.input.driver`: JDBC input driver name
+* `jdbctogcs.input.table`: JDBC input table name
+* `jdbctogcs.input.partitioncolumn` (Optional): JDBC input table partition column name
```

Comment:
move all the optional arguments togather in the bottom of this list.

Reply:
done